iam_builder/iam_builder.py

```diff
@@ -91,6 +92,12 @@ def build_iam_policy(config: dict) -> dict:  # noqa: C901
             iam["Statement"].append(secrets_statement)
             iam["Statement"].extend(iam_lookup["decrypt_statement"])
 
+    if "secretsmanager" in config:
+        secretsmanager_read_only = get_secretsmanager_read_only_policy(
+            config["secretsmanager"]["read_only"]
```

Comment:
This is _slightly weird_ given that we don't have a corresponding `write` access level. We should have specific error handling for if users try and pass in the same keys they do for S3 (e.g. `write_only`, `read_write` that throw errors to the end user that these levels of access have not been implemented in `iam_builder`.